packages/cli/src/commands/validator/list.ts

```diff
@@ -7,25 +9,43 @@ export default class ValidatorList extends BaseCommand {
 
   static flags = {
     ...BaseCommand.flags,
+    'no-truncate': flags.boolean({
+      description: "Don't truncate fields to fit line",
+      required: false,
+    }),
   }
 
   static examples = ['list']
 
   async run() {
-    this.parse(ValidatorList)
+    const res = this.parse(ValidatorList)
 
     cli.action.start('Fetching Validators')
+    const accounts = await this.kit.contracts.getAccounts()
     const validators = await this.kit.contracts.getValidators()
     const validatorList = await validators.getRegisteredValidators()
 
+    var names = []
+    for (let i = 0; i < validatorList.length; i++) {
+      names.push(accounts.getName(validatorList[i].address))
+    }
+    for (let i = 0; i < validatorList.length; i++) {
+      const name = await names[i]
+      validatorList[i].name = name ? name : ''
```

Comment:
Would it be easier to modify the `Validators` contractkit wrapper to pull the name? If we're going to want to the name everywhere we pull validator info, that might save us from having to duplicate this code elsewhere

Reply:
Sounds good